src/templates/layouts/eventListing/index.tsx

```diff
@@ -0,0 +1,82 @@
+/**
+ * ### Overview
+ * Event Listing represents the collection pages for Events within a facility.
+ *
+ * Event Listing expects data of type {@link FormattedEventListing}.
+ *
+ * ### Examples
+ * @see https://va.gov/butler-health-care/events/
+ *
+ */
+import { useEffect } from 'react'
+import { EventListing as FormattedEventListing } from '@/types/formatted/eventListing'
+import { EventWidgetTeaser } from '@/types/formatted/event'
+import { SideNavMenu } from '@/types/formatted/sideNav'
+import { ContentFooter } from '@/templates/common/contentFooter'
+import { LovellStaticPropsResource } from '@/lib/drupal/lovell/types'
+import { LovellSwitcher } from '@/templates/components/lovellSwitcher'
+
+// Allows additions to window object without overwriting global type
+interface customWindow extends Window {
+  sideNav?: SideNavMenu
+  pastEvents?: {
+    entities: EventWidgetTeaser[]
+  }
+}
+declare const window: customWindow
+
+export function EventListing({
+  title,
+  id,
+  introText,
+  events,
+  menu,
+  lovellVariant,
+  lovellSwitchPath,
+}: LovellStaticPropsResource<FormattedEventListing>) {
+  // Add data to the window object for the sidebar & event widgets
+  useEffect(() => {
+    window.sideNav = menu
+
+    // This populates the whole events widget, even upcoming events. IDK!
+    window.pastEvents = { entities: events }
+  }, [menu, events])
```

Comment:
I'm not sure this is enough. In content-build, both `pastEvents` and `allEventTeasers` are populated. However, it's not clear if that is actually working correctly either. See [this Slack discussion.](https://dsva.slack.com/archives/C01SR56755H/p1705100651129049?thread_ts=1705100611.180519&cid=C01SR56755H)

Reply:
I think this is working correctly in that the functionality of the filters matches what prod does. The list appears to filter to the same results whether `allEventTeasers` is included or not. Open to opinions otherwise, but again, I think refactoring the Events Listing widget from vets-website is out of scope for our team.